src/base/settingsstorage.cpp

```diff
@@ -26,7 +26,9 @@
  * but you are not obligated to do so. If you do not wish to do so, delete this
  * exception statement from your version.
  */
+#include "settingsstorage.h"
```

Comment:
Placing the base header first in the list of includes gives a good way to check that the header contain all the required includes and/or forward declarations to be compiled without errors.